free-1.md

```diff
@@ -4,47 +4,143 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The project would serve people interacting with html and do web development. I
+think a language that can streamline the writing of html and its interaction
+with javascript and css would greatly benefit people that use these
+technologies. There currently are a few (many) javascript modules and libraries
+that try to do this. For example, jquery makes it easier to manipulate html on
+a webpage. However, as we saw in class, jquery's syntax was very verbose and
+at many points confusing. It has become bloated to become a large library. Most
+casual users who just want to build a simple website do not need jquery.
+
+I am looking more at something like JSX (https://facebook.github.io/jsx/).
+However, JSX only officially works within the React framework. I like JSX due to
+its clean syntax. If a DSL that uses JSX's syntax or something similar to
+interact directly with how to write HTML and javascript for a page, then it
+would be very nice.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+experience like now? What would their experience be like if you could help
 them?_
 
+Writing HTML, CSS, and javascript is often more verbose than it needs to be,
+often very repetitive (all the closing tags and so forth). People often have to
```

Comment:
Many times the closing tags feel extraneous; additionally, there are a surprising number of tags for whom the HTML5 spec requires closing tags (although the browser will accept it anyway). If you don't need extra visual cruft, then don't use it. There are a lot of cool template libraries that generate HTML from things like

```
div attr1=val, attr2=otherval
    this is the contents of that div
    this is super exciting
    but I don't need a closing tag because scope is indentation controlled
```